src/frontend/frontend.ts

```diff
@@ -151,6 +158,11 @@ class App {
       this.searchDisabled = false;
     }
   }
+
+  themeChanged(newTheme) {
+    this.theme = newTheme;
+    // TODO: do stuff here to persist the theme selection
```

Comment:
Does this mean I will have to re-select dark theme each time I reopen Chrome / Augury?